src/components/fields/async-select-field/async-select-field.visualspec.js

```diff
@@ -0,0 +1,13 @@
+import { percySnapshot } from '@percy/puppeteer';
+import { widths } from '../../../../test/percy/widths';
+
+describe('AsyncSelectField', () => {
+  beforeAll(async () => {
+    await page.goto('http://localhost:3001/async-select-field');
```

Comment:
1. Can‘t we just return the promise here instead of having the beforeAll being async?
2. What if we have a gotoComponent exported from the utils somewhere to not repeat localhost and ports here over and over again?

Reply:
gotoRoute might be more appropriate as route specs export a route.

Reply:
I added HOST to the jest globals instead [here](https://github.com/commercetools/ui-kit/pull/373/commits/4abd35969cf95d7076e19c2d229c8bb02ff099c4)